composer.json

```diff
@@ -24,50 +25,17 @@
   ],
   "license": "LGPL-3.0",
   "require": {
-    "php": ">= 5.5.0, <= 7.1.99",
-    "typo3/cms": ">= 7.6.0, <= 8.7.99",
-    "pelago/emogrifier": "^1.2"
+    "php": ">= 5.5.0, <= 7.3.99",
+    "typo3/cms-core": ">= 7.6.0, <= 9.5.99",
```

Comment:
TYPO3 v7 is not maintained anymore (except ELTS but I don't care).

Point is the dependency to `typo3/cms-core` is wrong for TYPO3 v7 since it does not exist there. Please see the tags on https://packagist.org/packages/typo3/cms-core

Next, having the dependency to v8 and v9 only means PHP dependency should be changed to at least 7.0

Reply:
You're right, I forgot about the missing versions of cms-core.